src/theme/theme.ts

```diff
@@ -145,6 +145,17 @@ export const themeWME = {
         fontFamily,
       },
     },
+    MuiTooltip: {
```

Comment:
I had to put the styling for the Popper in the tooltip here since it's not a direct child of `MuiTooltip`.